infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -1804,5 +1804,31 @@
     },
     "dropdown": {
         "add-new": "Lisää uusi"
+    },
+    "privilege": {
```

Comment:
Oikeudet löytyvät nyt täältä kutakin oikeutta vastaavan nimen takaa. Toim.huom. sen lisäksi että siirsin nämä tänne, niin muutin myös joidenkin oikeuksien käännöksiä (ainakin omasta mielestäni) selkeämpään suuntaan. Mm. "luku"-termi on vaihdettu "tarkastelu"-termiin (omasta mielestäni selkeämpi näin,) ja joitain käännöksiä on muutenkin pidennetty (aiemmin kantacolumnin maksimipituus kinnasi tätä vastaan,)